main.go

```diff
@@ -19,12 +19,13 @@ func main() {
 	usage := `zabbix-agent-extension-rabbitmq
 
 Usage:
-    zabbix-agent-extension-rabbitmq [-r <address>] [-u <name>] [-s <password>] [-c <path>] [-z <host>] [-p <number>] [-o <name>] [-d [-g <name>] [-a]]
+    zabbix-agent-extension-rabbitmq [-r <address>] [-u <name>] [-s <password>] [-t <timeout>] [-c <path>] [-z <host>] [-p <number>] [-o <name>] [-d [-g <name>] [-a]]
 
 RabbitMQ options:
     -r --rabbitmq <address>          Listen address of RabbitMQ server [default: http://127.0.0.1:15672]
     -u --rabbitmq-user <name>        RabbitMQ management username [default: guest]
     -s --rabbitmq-secret <password>  RabbitMQ management password [default: guest]
+    -t --rabbitmq-timeout <timeout>  RabbitMQ request timeout in ms [default: 5000]
```

Comment:
You can set a default value as `5000ms` and parse it through `time.ParseDuration`.

Reply:
Changed to 5000ms. Thanks.